client/src/api/requests/scenarios.ts

```diff
@@ -37,3 +37,31 @@ export class GetScenario extends Request<Scenario> {
     return data as unknown as Scenario
   }
 }
+
+export class UpdateScenarioConfig extends Request<{result: string; message: string}> {
```

Comment:
In our Gitlab code for webconsole we have a similar request called `UpdateScenario`. I think for consistency sake it's better to name this one `UpdateScenario` as well